packages/react-refresh/src/__tests__/ReactFresh-test.js

```diff
@@ -1305,7 +1313,7 @@ describe('ReactFresh', () => {
 
   it('can patch both trees while suspense is displaying the fallback', async () => {
```

Comment:
This test is no longer passing. Seems like it really can't patch both trees anymore. @gaearon Could you take a look if there's a legit bug or something we just can't test in new roots anymore.